src/test/java/uk/gov/hmcts/reform/profilesync/util/UserProfileSyncJobSchedulerTest.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.hmcts.reform.profilesync.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.time.LocalDateTime;
+
+import org.junit.Test;
+import uk.gov.hmcts.reform.profilesync.advice.UserProfileSyncException;
+import uk.gov.hmcts.reform.profilesync.domain.SyncJobAudit;
+import uk.gov.hmcts.reform.profilesync.domain.SyncJobConfig;
+import uk.gov.hmcts.reform.profilesync.repository.SyncConfigRepository;
+import uk.gov.hmcts.reform.profilesync.repository.SyncJobRepository;
+import uk.gov.hmcts.reform.profilesync.schedular.UserProfileSyncJobScheduler;
+import uk.gov.hmcts.reform.profilesync.service.ProfileSyncService;
+
+
+public class UserProfileSyncJobSchedulerTest {
```

Comment:
Please add any missing `verify()` to the `when()` calls

Reply:
done